examples/tracer-web/examples/fetch/index.js

```diff
@@ -31,10 +30,11 @@ registerInstrumentations({
 
 const webTracerWithZone = provider.getTracer('example-tracer-web');
 
+// eslint-disable-next-line no-undef
```

Comment:
If the no-undef rule is disabled for web APIs then we should add the web environment to the examples' eslint config? https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments